data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,470 @@
+		payment 10000000
+		conversation
+			`You finally dock at <planet>, your tedious tour complete. Shortly after, you receive a ping informing you that <payment> has been deposited into your account. Terry must've set it up to happen automatically.`
+				decline
+
+
+mission "ZZ Hai Building Mountaintop [2a] Sales Job"
+	landing
+	invisible
+	source 
+		not planet "Hai-home"
+		not planet "Mountaintop"
+		not planet "Darkwaste"
+		government "Hai"
+	to offer
+		has "Hai Building Mountaintop [2] Recruits: active"
+	on offer
+		"hai dog and pony" ++
+		conversation
+			branch overdone
+				"hai dog and pony done" >= 1
+			branch toomany
+				"hai dog and pony" >= 17
+			branch interminable
+				"hai dog and pony" >= 14
+			branch ordeal
+				"hai dog and pony" >= 11
+			branch tiresome
+				"hai dog and pony" >= 8
+			branch tedious
+				"hai dog and pony" >= 5
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop.`
+				defer
+			label tedious
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. Despite the enthusiasm of the Hai it's tediously repetitive by now.`
+				defer
+			label tiresome
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. The routine is well-practiced now, but starting to get a little tiresome.`
+				defer
+			label ordeal
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. You struggle to maintain a cheerful and enthusiastic facade as you count down the minutes until this ordeal is over.`
+				defer
+			label interminable
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. Why did you ever agree to this interminable dog and pony show? Is it really worth all this just to save one tiny little galaxy?`
+				defer
+			label toomany
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. You're pretty sure you've done this more times than there are habitable planets in Hai space. Did you lose track and repeat a world? Will this nightmare ever end?`
+				defer
+			label overdone
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. Terry makes good on her promise of forcing you to shuffle out and do some last-minute spruiking to offload the remaining promotional materials to those who missed it the first time.`
+				decline
+
+
+
+mission "Hai Building Mountaintop [3] Furnishings"
+	name "Retrieve human furnishings."
+	description "The volunteers are so numerous that the Hai need outside help transporting furnishings for their quarters to the new station."
+	autosave
+	landing
+	source "Mountaintop"
+	stopover "Prime"
+	destination "Mountaintop"
+	clearance
+	on visit
+		dialog `You have arrived at the Mountaintop construction site, but it isn't finished or staffed yet.`
```

Comment:
This doesn't seem like a useful stopover message.

Reply:
You only get this if you try and skip stops in Hai space.